src/modules/provisioning/tenant-options-provisioning/tenant-options-provisioning.component.ts

```diff
@@ -25,10 +26,16 @@ export class TenantOptionsProvisioningComponent {
     private alertService: AlertService,
     private provisioning: ProvisioningService,
     public datasource: TenantOptionsTableDatasourceService,
-    private tenantSelectionService: TenantSelectionService
+    private tenantSelectionService: TenantSelectionService,
+    private tenantService: TenantService
   ) {
     this.columns = this.getDefaultColumns();
   }
+  ngOnInit(): void {
+    this.tenantService.current().then((tenant) => {
+      this.tenant = tenant.data;
+    });
+  }
```

Comment:
If you nee the current tenant, please use [the AppStateService](https://resources.cumulocity.com/documentation/websdk/ngx-components/injectables/AppStateService.html#currentTenant) instead of performing an additional request..